tests/test_core.py

```diff
@@ -4566,6 +4566,9 @@ def test_fs_trackingdelegate(self):
 
   @also_with_noderawfs
   def test_fs_writeFile(self, js_engines=None):
+    if self.is_osx() and 'NODERAWFS=1' in self.emcc_args:
```

Comment:
Too bad to miss reviewing #6128, but can we rename `is_osx` to `is_macos`, as Apple renamed OS X to macOS to keep consistency with iOS, watchOS and tvOS?

Reply:
Done in #6138.